test/unit-tests/cve/cveUpdateTest.js

```diff
@@ -81,12 +89,22 @@ describe('Testing the PUT /cve/:id endpoint in Cve Controller', () => {
       }
       class CveRepo {
         async updateByCveId (cveId, newCve) {
-          return { n: 0 }
+          // test shouldn't reach here
+          expect.fail()
+          return null
+        }
+
+        async findOneByCveId () {
+          return null
         }
       }
       class CveIdRepo {
         async updateByCveId (cveId, newCve) {
-          return { n: 1 }
+          return null
```

Comment:
I think that along with the fail statement in CveRepo, we should also put it in CveIdRepo because both documents shouldn't be changed in this scenario and that would cover our bases.
```suggestion
          expect.fail()
          return null
```